external-docs/docs/guides/configuration/templates.md

```diff
@@ -31,41 +31,31 @@ In the above example, we have reused the `year` template, but given the dimensio
 
 ## Available templates
 
-The templates are currently in version 1.0; with future development planned. You can see the entire list of the templates inline below or on [csvcubed github](https://github.com/GSS-Cogs/csvcubed/tree/main/src/csvcubed/readers/cubeconfig/v1_0/templates). The `from_template` functionality fetches the most recent version of our templates from the web.
+The templates are currently in version 1.0; with future development planned. You can see the entire list of the templates inline below or on [csvcubed github](https://github.com/GSS-Cogs/csvcubed/tree/main/src/csvcubed/readers/cubeconfig/v1_0/templates). The `from_template` funcitonality fetches the most recent version of our templates from the web.
```

Comment:
You've introduced a typo :P

I don't think you should be making any changes at all in this file in this PR.